cmd/osde2ectl/get/cmd.go

```diff
@@ -92,6 +106,7 @@ func run(cmd *cobra.Command, argv []string) error {
 		if err != nil {
 			return fmt.Errorf("Error getting the cluster's kubeconfig - %s", err)
 		}
+		fmt.Printf("Cluster kubeconfig info - %v", config.Kubeconfig.Contents)
```

Comment:
We shouldn't print out the kubeconfig contents to stdout/err. It should get written to a file. We don't want kubeconfig contents accidentally getting logged to a public place if this is used in CI.

I could see some merit to maybe doing it if we support setting a debug output, but at the moment it should be omitted.